tests/test-recipes/build_recipes.sh

```diff
@@ -31,8 +31,19 @@ echo "$OUTPUT" | grep "Error" | wc -l | grep 6
 ! OUTPUT=$(conda build --no-anaconda-upload conda-meta/ 2>&1)
 echo "$OUTPUT" | grep 'Error: Untracked file(s) ('\''conda-meta/nope'\'',)'
 
+# Get the version of conda as the error message changed in conda 4.
+set +e
+echo $(conda -V 2>&1) | grep "conda 3\..*"
+export IS_CONDA_3=$?
```

Comment:
Switched to treat conda 3 as the special case instead of conda 4. The effect is the same.